core/src/fight/backgrounds.ts

```diff
@@ -1,30 +1,35 @@
+export enum MapEffect {
+	HalfSpeed,
+}
+
 export type FightBackground = {
   name: string,
   odds: number,
+  effect?: MapEffect,
 };
 
 export const tournamentBackground = {
   name: '3.png',
-  odds: 0,
+  odds: 0
 };
 
 export const bossBackground = {
   name: '4.png',
-  odds: 0,
+  odds: 0
 };
 
 export const fightBackgrounds: FightBackground[] = [
-  { name: '1.jpg', odds: 100 },
-  { name: '2.jpg', odds: 2 },
+  { name: '1.jpg', odds: 100},
+  { name: '2.jpg', odds: 2},
   tournamentBackground,
   bossBackground,
-  { name: '5.png', odds: 2 },
-  { name: '6.png', odds: 2 },
-  { name: '7.png', odds: 2 },
-  { name: '8.png', odds: 2 },
-  { name: '9.png', odds: 2 },
-  { name: '10.png', odds: 2 },
-  { name: '11.png', odds: 2 },
-  { name: '12.png', odds: 2 },
-  { name: '13.png', odds: 2 },
+  { name: '5.png', odds: 2},
+  { name: '6.png', odds: 2},
+  { name: '7.png', odds: 2},
+  { name: '8.png', odds: 2},
+  { name: '9.png', odds: 2},
+  { name: '10.png', odds: 1, effect: MapEffect.HalfSpeed},
+  { name: '11.png', odds: 2},
+  { name: '12.png', odds: 1},
```

Comment:
No need to lower the odds if no modifier

```suggestion
  { name: '12.png', odds: 2},
```